docs/content/en/docs/reference/packagespec/harbor/v2.5.1.md

```diff
@@ -0,0 +1,59 @@
+| `internalTLS.enabled`                                                | Enable TLS for the components (core, jobservice, portal, registry)                                                                                                                                                                                                                                                                                                                                                                            | `true`                               |
+| **Persistence**                                                      |                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |                                       |
+| `persistence.resourcePolicy`                                         | Setting it to `keep` to avoid removing PVCs during a helm delete operation. Leaving it empty will delete PVCs after the chart deleted. Does not affect PVCs created for internal database and redis components.                                                                                                                                                                                                                                                   | `keep`                                |
+| `persistence.persistentVolumeClaim.registry.size`                    | The size of the volume                                                                                                                                                                                                                                                                                                                                                                                                                                            | `5Gi`                                 |
+| `persistence.persistentVolumeClaim.registry.storageClass`            | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used (the default). Set it to `-` to disable dynamic provisioning                                                                                                                                                                                                                                                                                                    |                  ""                     |
```

Comment:
```suggestion
| `persistence.persistentVolumeClaim.registry.storageClass`            | Specify the `storageClass` used to provision the volume, or the default StorageClass will be used (the default). Set it to `-` to disable dynamic provisioning                                                                                                                                                                                                                                                                                                    |                  ""                     |
```
I'm not sure why you have "(the default)" there in parenthesis. Is that a literal value.